app.py

```diff
@@ -87,10 +93,21 @@
     ],
 )
 
+# Generate an OIDC provider, allowing CI workers to assume roles in the account
+
+stack.add_oidc_provider(
```

Comment:
can we make this optional because some implementations may not have OIDC setup? Something like (I'm sure this won't run and it's not the only change needed to make the OIDC provider optional, just conveying the thought)
```
oidc_thumbprint := config.oidc_thumbprint
oidc_provider_url := config.oidc_provider_url
if oidc_thumbprint and oidc_provider_url:
   stack.add_oidc_provider
...
```